src/syngen/ml/config/configurations.py

```diff
@@ -62,5 +62,5 @@ def set_paths(self):
             "tmp_store_path": f"model_artifacts/tmp_store/{dynamic_name}",
             "vae_resources_path": f"model_artifacts/resources/{dynamic_name}/vae/checkpoints/",
             "dataset_pickle_path": f"model_artifacts/resources/{dynamic_name}/vae/checkpoints/model_dataset.pkl",
-            "fk_kde_path": f"model_artifacts/resources/{dynamic_name}/vae/checkpoints/fk_kde.pkl"
+            "fk_kde_path": f"model_artifacts/resources/{dynamic_name}/vae/checkpoints/"
```

Comment:
I guess it should be `model_artifacts/resources/{dynamic_name}/vae/checkpoints/fk_kde`
Otherwise we have names like "_parentcustomerid.pkl"
Or we could add "fk_kde" to all paths